docs/src/convert-to-onda.md

```diff
@@ -199,8 +200,57 @@ If any errors _were_ encountered, you may need to iterate further.
-For storing the plan, use [`OndaEDF.write_plan`](@ref) (or `Legolas.write(file_path, plan, FilePlanV3SchemaVersion())` (see the documentation for [`Legolas.write`](https://beacon-biosignals.github.io/Legolas.jl/stable/#Legolas.write) and [`FilePlanV3`](@ref OndaEDF.FilePlanV3).
+Note that `Onda.store` requires at a minimum the file path (or "path-like" writeable), the format to use to serialize the samples (like `"lpcm"` or `"lpcm.zst"`), and the `Samples` themselves.
+However, this only _serializes_ the samples to the file path.
+In most cases, you will want a _signal record_ that can be stored elsewhere, which combines the path and file format to the serialized samples along with the `SamplesInfo` and other metadata required to be able to _deserialize_ and use these samples.
+In this case, you should use the second `Onda.store` method which additionally requires:
+
+- `recording::UUID` (the recording that this signal is associated with)
+- `start::Period` (the start time of this signal _relative to the start of the recording_)
+- `sensor_label::AbstractString` (a _unique label_, defaults to the `sensor_type`)
+
+While the `edf_to_onda_samples` returns an `OrderedDict` whose keys are valid `sensor_label`s when considering the EDF file alone, Onda requires that `sensor_label` be unique for the entire _recording_.  If you have signals from other sources that are associated with the same recording (e.g., different devices, multiple EDF files, etc.) that may have common `sensor_type`s, you must ensure that the `sensor_label`s that you store the signals with are _globally_ unique.
+
+#### Example: storing samples
+
+Consider a case with parallel recordings from a traditional EEG system (received as an EDF) and some custom EEG hardware (which emits Onda-formatted data natively).
+Each device generates multichannel EEG data and uses `sensor_type="eeg"`.
+We must adjust the generated sensor labels for the EDF signals before storing them so that they are unique, which we can do by prepending `"edf_"` to the default, `sensor_type`-derived labels before storing:
+
+```julia
+recording = uuid4()
+prototype_signals = ingest_prototype_device_as_onda(prototype_path, recording)
+
+edf = EDF.File(edf_path)
+plans = map(plan_edf_to_onda_samples(edf)) do plan
+    Legolas.record_merge(plan; recording, sensor_label="edf_" * plan.sensor_label)
+end
+
+edf_samples, plan_executed = edf_to_onda_samples(edf, plans)
+
+start_relative_to_recording = Second(0)
+edf_signals = map(keys(edf_samples)) do sensor_label
+    samples = edf_samples[sensor_label]
+    return Onda.store(joinpath("samples", sensor_label * ".lpcm"), "lpcm", samples,
+                      recording, start_relative_to_recording, sensor_label)
```

Comment:
is there some way to get at the plan inside here? it feels unsatisfying if we only fix the sensor label thing without fixing the mismatch that we have 2 paired outputs but you can't associate them with each other

Reply:
I think the way to do it now is filter `plan_executed` to matching sensor labels. which is alright, at least we can do it now